modules/ROOT/pages/basic-alerts.adoc

```diff
@@ -40,4 +40,79 @@ The following levels of severity are available for basic alerts:
 
 include::partial$include-alert-severity.adoc[leveloffset=+1]
 
-// add basic alert email templates
+[[alert-variables]]
+== Basic Alert Email Template
+
+You can use the following variables in a basic alert email's *Subject* and *Message* fields.
+For the description of each variable, see <<var-defs>>.
+
+[%header,cols="25a,50a,20a"]
+.Variables for Alerts
+|===
+| Alert Type | Alert | Applicable Variables
+2+| All Alerts | `${resource}` `${severity}` `${state}`
+.7+| CloudHub App Alerts |*CPU usage - CloudHub* .2+|`${operator}` `${value}`
+| *Memory usage - CloudHub*
```

Comment:
For cloudhub apps we support these metrics.

Cpu Utilization
Memory Utilization
Thread Count
Message Count
Message Error Count
Message Response Time